examples/daal/cpp_sycl/source/utils/service.h

```diff
@@ -507,12 +507,12 @@ void printNumericTables(NumericTable * dataTable1, NumericTable * dataTable2, co
     {
         for (size_t j = 0; j < nCols1; j++)
         {
-            std::cout << std::setw(interval) << std::setiosflags(std::ios::fixed) << std::setprecision(3);
+            std::cout << std::setw(interval) << std::setiosflags(std::ios::fixed) << std::setprecision(5);
```

Comment:
Why you changed precision for all examples?

Reply:
Cause it led to the wrong output

Reply:
Fixed. Precision template parameters were added